db/db_flush_test.cc

```diff
@@ -78,6 +79,7 @@ TEST_F(DBFlushTest, SyncFail) {
   // Flush job should release ref count to current version.
   ASSERT_EQ(refs_before, cfd->current()->TEST_refs());
   Destroy(options);
+#endif  // ROCKSDB_LITE
```

Comment:
Thanks for fixing it for me! Can we comment out just the `FilesPerLevel` call instead of the whole test?